### ISSUE TITLE
Unclear what "Play" button does

### ISSUE THREAD

**Issue by Bios-Marcel** [scribble-rs/ScribblersPad#25]
There's currently "Play", "Create New" and "Find". While find is obvious, play isn't.
Maybe we should call it "Quick play", "Create Lobby", "Find Lobby". While verbose, it would make things a bit more clear.

**Comment by BigETI** [comment]
Those labels are placeholders for images. Since we do not have any proper artworks, they have been labeled that way to at least understand what their functions are.

**Comment by Bios-Marcel** [comment]
It'll be hard to find fitting images for things like that I think

**Comment by BigETI** [comment]
Correct, therefore I prefer them to be specifically made for this game according to #23.
